src/components/standalone/hotspot/SettingsContent.vue

```diff
@@ -0,0 +1,695 @@
+        id: item.id,
+        label: item.name
+      }))
+      configurationForm.value.parentHotspot = getParentHotspot.data.parents[0].id
+    }
+  } catch (exception: any) {
+    errorLoadingData.value.notificationTitle = t('error.cannot_retrieve_parent_hotspot')
+    errorLoadingData.value.notificationDescription = t(getAxiosErrorMessage(exception))
+  } finally {
+    try {
+      let getNetworkDevice = await ubusCall('ns.dedalo', 'list-devices', {})
+      if (
+        getNetworkDevice &&
+        getNetworkDevice.data &&
+        getNetworkDevice.data.devices &&
+        getNetworkDevice.data.devices.length
+      ) {
+        networkDeviceList.value = getNetworkDevice.data.devices.map((item: NetworkDevice) => ({
+          id: item,
+          label: item
+        }))
+        configurationForm.value.networkDevice = getNetworkDevice.data.devices[0]
+      } else emptyDevices.value = true
+    } catch (exception: any) {
+      errorLoadingData.value.notificationTitle = t('error.cannot_retrieve_network_device')
+      errorLoadingData.value.notificationDescription = t(getAxiosErrorMessage(exception))
+    } finally {
+      loading.value = false
+    }
+  }
+}
+
+async function getConfiguration() {
+  try {
+    let getDataConfiguration = await ubusCall('ns.dedalo', 'get-configuration', {})
+    if (
+      getDataConfiguration &&
+      getDataConfiguration.data &&
+      getDataConfiguration.data.configuration &&
+      getDataConfiguration.data.configuration.connected
+    ) {
+      activeConfiguration.value = getDataConfiguration.data.configuration.hotspot_id != ''
+      let configuration = getDataConfiguration.data.configuration
+      configurationForm.value.parentHotspot = configuration.hotspot_id
+      configurationForm.value.unitName = configuration.unit_name
+      configurationForm.value.unitDescription = configuration.unit_description
+      configurationForm.value.networkDevice = configuration.interface
+      if (configuration.network) {
+        configurationForm.value.networkAddress = configuration.network
+        getDhcpRange()
+      }
+      if (configuration.dhcp_start)
+        configurationForm.value.dhcpRangeStart = configuration.dhcp_start
+      if (configuration.dhcp_end) configurationForm.value.dhcpRangeEnd = configuration.dhcp_end
+    } else isLoggedIn.value = false
+  } catch (exception: any) {
+    errorLoadingData.value.notificationTitle = t('error.cannot_retrieve_configuration')
+    errorLoadingData.value.notificationDescription = t(getAxiosErrorMessage(exception))
+  }
+}
+
+function validateLogin(): boolean {
+  let isValidationOk = true
+  let isFocusInput = false
+
+  if (!loginForm.value.hostname) {
+    let { valid, errMessage } = validateRequired(loginForm.value.hostname)
+    if (!valid) {
+      error.value.hostname = t(errMessage as string)
+      isValidationOk = false
+    }
+  }
+
+  if (!isValidationOk) {
+    focusElement(hostnameRef)
+    isFocusInput = true
```

Comment:
`focusElement(...)` and `isFocusInput = true` can be moved under the previous `isValidationOk = false`